### ISSUE TITLE
"Enable hardware acceleration" not work

### ISSUE THREAD

**Issue by LuckyTil** [a85/POSTMan-Chrome-Extension#11125]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

"Enable hardware acceleration" not work

### Steps To Reproduce

Help - "Enable hardware acceleration" - Nothing happens

### Operating System

Windows 10 x64

### Postman Version

9.26.3

### Postman Platform

Postman App

**Comment by DannyDainton** [comment]
Hey @LuckyTil

Could you elaborate more on the issue your facing, please?

What's not working for you? How are you measuring the before and after effect when enabling this feature?

Screenshots or a short video will help us to see the issue.

**Comment by LuckyTil** [comment]
Hi, Danny.

> Could you elaborate more on the issue your facing, please?

I can switch option from "Disable hardware acceleration" to "Enable hardware acceleration" state.
But I can't switch option back.


Ваш лист від 28.07.2022, 23:25:22:

> Hey @LuckyTil
> Could you elaborate more on the issue your facing, please?
> What's not working for you? How are you measuring the before and after effect when enabling this feature?
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.

-- 
З повагою,
Lucky.Til
***@***.***

**Comment by DannyDainton** [comment]
It's an issue with enabling the option, once you had disabled it? Rather than the whole feature not actually working? 

When you disable the option, does it tell you to restart Postman? Once it's been restarted, you can no longer enable it?

**Comment by LuckyTil** [comment]
Hi, Danny Dainton.

> It's an issue with disabling the option, once you had enabled it? Rather than the feature not actually working?

I don't know if this feature actually works.
I see that after selecting the "Enable hardware acceleration" option in the menu, its value remains unchanged.






-- 
 З повагою,
 Lucky.Til
***@***.***

**Comment by jviotti** [comment]
Hi @LuckyTil ,

What GPU brand/model do you have?

**Comment by LuckyTil** [comment]
Hi, Juan.

> What GPU brand/model do you have?

1. Intel UHD 630
2. GeForce RTX 2060

-- 
З повагою,
Lucky.Til
***@***.***

**Comment by simepy** [comment]
Same situation with GPU: Intel® HD Graphics 620 on Archlinux with `Postman 9.31.0, linux-5.19.6-arch1-1 on x64`

**Comment by starlight173** [comment]
Once I disable it, I cannot re-enable. It happens both on Linux and Mac. I dont have Windows.

**Comment by abdullahwaqar** [comment]
Issue still persists

```
Postman for Linux

Version 10.6.7

UI Version 10.6.7-ui-221210-1433

Desktop Platform Version 10.6.0

Architecture x64

OS Platform linux 6.0.12-1-default
```

**Comment by user414** [comment]
I can also confirm that it is still is not working on linux with version 10.6.7. Unfortunately, this settings does not seem to be in the settings.json file or the Preferences file. However, a workaround is to export your data from the settings, delete your postman folder in .config, restart the app and then import your data. After that hardware acceleration will be enabled again.

**Comment by zhidongnick** [comment]
@DannyDainton, I have the same problem and I am using MacBook Pro M1 Max. After I disable hardware acceleration, I cannot enable hardware acceleration again. Currently, I upgrade my Postman to Version 10.8.7. This bug is still not fixed.

**Comment by bavulapati** [comment]
We have landed the fix internally. The fix will be available from the version 10.10.0

**Comment by bavulapati** [comment]
The fix is available to all users
